package-examples/helloworld-kustomize/README.md

```diff
@@ -0,0 +1,22 @@
+helloworld-kustomize
+==================================================
+
+# NAME
+
+  helloworld-kustomize
+
+# SYNOPSIS
+
+  kubectl apply --recursive -f helloworld-kustomize
+
+# Description
+
+a sample where the upstream package contains a kustomize patch that contains a setter.  One can run the pipeline in the package
+```
+kpt fn render .
+```
+and observe the message in the package change.  You can also see documentation on `apply-setters` function to see a full range
```

Comment:
if we're going to keep this sample I think the description can be improved, maybe more specifically explain what message changes and how it is changed

Reply:
I think right now I am separating the work of updating the format and reconciling the intent of these samples.  You are correct that it doesn't really make a ton of sense because the overall story of what we are trying to say is not clear.